src/crypto/crypto_util.cc

```diff
@@ -106,7 +106,7 @@ int NoPasswordCallback(char* buf, int size, int rwflag, void* u) {
 }
 
 void InitCryptoOnce() {
-#ifndef OPENSSL_IS_BORINGSSL
+#if !defined(OPENSSL_IS_BORINGSSL) && OPENSSL_VERSION_MAJOR < 3
```

Comment:
We don't need to call `OPENSSL_init_ssl` with OpenSSL 3?

Reply:
Sorry you are right we do need to call `OPENSSL_int_ssl`, that is a mistake and should only be skipping the setting of the config_filename for OpenSSL 3 as that will already have been set.